EpikCord/client/client_application.py

```diff
@@ -106,53 +104,54 @@ async def edit_global_application_command(
         name: Optional[str] = None,
         description: Optional[str] = None,
         options: Optional[List[AnyOption]] = None,
-        default_permissions: Optional[bool] = None,
+        default_member_permissions: Optional[Permissions] = None,
     ):
-        payload = {}
-        if name:
-            payload["name"] = name
-        if description:
-            payload["description"] = description
+        payload: Dict[str, Optional[Union[List[discord_typings.ApplicationCommandOptionData], str, int]]] = {
+            "name": name,
+            "description": description,
+        }
         if options:
             payload["options"] = [option.to_dict() for option in options]
-        if default_permissions:
-            payload["default_permissions"] = default_permissions
+        if default_member_permissions:
+            payload["default_member_permissions"] = default_member_permissions.value if isinstance(default_member_permissions, Permissions) else default_member_permissions
 
         await self.client.http.patch(
             f"/applications/{self.id}/commands/{command_id}", json=payload
         )
 
-    async def delete_global_app_command(self, command_id: str):
+    async def delete_global_application_command(self, command_id: str):
         await self.client.http.delete(f"/applications/{self.id}/commands/{command_id}")
 
-    async def bulk_overwrite_global_app_commands(self, commands: List[Dict]):
+    async def bulk_overwrite_global_application_commands(self, commands: List[Dict]):
         await self.client.http.put(f"/applications/{self.id}/commands", json=commands)
 
-    async def fetch_guild_app_commands(self, guild_id: str):
+    async def fetch_guild_application_commands(self, guild_id: str):
         response = await self.client.http.get(
             f"/applications/{self.id}/guilds/{guild_id}/commands"
         )
         return [ApplicationCommand(command) for command in await response.json()]
 
-    async def create_guild_app_command(
+    async def create_guild_application_command(
```

Comment:
not sure why `app` as been reverted to `application` it makes the method name way too long

Reply:
I just like `Application` over `App`

Reply:
Maybe aliases?